main.tf

```
@@ -93,6 +93,7 @@ resource "azurerm_firewall_policy_rule_collection_group" "this" {
       dynamic "rule" {
         for_each = var.bastion_config.ipg_rdp_access_ids
         content {
+          # Generate rule name from the IP group name to avoid conflicts. The name is captured from the resource ID
```

```suggestion
          # The regex outputs the name of the ip group from id.
```
Is that correct? The explanation is not really helpful :D